frontend/src/components/SettingsGeneral.vue

```diff
@@ -14,7 +14,7 @@ user.init(call);
 // handle ui languages
 watch(locale, (newValue: string) => {
   user.data.settings.language = newValue;
-  window.location.reload(); // This is needed to reload locale for dayjs instance
+  // This is needs a window.location.reload(); to reload locale for dayjs instance
```

Comment:
tiny nit, typo: "this `is` needs a"

Reply:
Oh thanks for highlighting this one. I commented this out because this currently causes an issue of the language setting not being saved to db because the reload triggers too early. I'll fix that in another PR.